### ISSUE TITLE
[BUG]: element_scrollable_tabbable violation is still seen with achecker tool.

### ISSUE THREAD

**Issue by psoujany** [IBMa/equal-access#2298]
### Project

accessibility-checker-engine

### Operating System

MacOS

### Automated testing tool and ruleset.

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

We’re observing `element_scrollable_tabbable` violation in our API docs. But we don’t see this violation when we do scan using Accessibility Assessment(plugin), only see this violation while using achecker tool. Earlier we used to observe this violation in all the docs, but now volume is reduced only few docs we see this violation.

### Steps to reproduce

Run achecker on the attaching .html using tool, many divs failed to get auto-focus. I've also attached the results. Thank you.

[re-create.zip](https://github.com/user-attachments/files/20547889/re-create.zip)

**Comment by wkeese** [comment]
@shunguoy - You closed this ticket as "completed" but [the rule](https://github.com/IBMa/equal-access/blob/main-4.x/accessibility-checker-engine/src/v4/rules/element_scrollable_tabbable.ts) is still there.

**Comment by psoujany** [comment]
@shunguoy Where is the rule to be turned off? Could you please tell us on doing that. Thank you.

**Comment by shunguoy** [comment]
@psoujany Sorry, this is a duplicate of #2281 and will be closed upon completion of #2281.
